src/main/java/com/stripe/model/EventDataObjectDeserializer.java

```diff
@@ -190,7 +192,8 @@ public StripeObject deserializeUnsafe() throws EventDataObjectDeserializationExc
   public StripeObject deserializeUnsafeWith(CompatibilityTransformer transformer) {
     return StripeObject.deserializeStripeObject(
         transformer.transform(rawJsonObject.deepCopy(), apiVersion, eventType),
-        this.responseGetter);
+        this.responseGetter,
+        ApiMode.V1);
```

Comment:
Where are the v2 counterparts to the changes in this file?

Reply:
This is a V1 only thing for now. 
https://github.com/stripe/stripe-java/blob/33589343406af111bc977877f7ab96313bbe5ada/src/main/java/com/stripe/model/EventDataObjectDeserializer.java#L40-L50

V2 uses [Eventtypeadapterfactory](https://github.com/stripe/stripe-java/blob/5cd2f1b3547be92ccf3d84292be0df9eee9bb41e/src/main/java/com/stripe/model/v2/EventTypeAdapterFactory.java#L11)